app/helpers/application_helper.rb

```diff
@@ -39,4 +39,8 @@ def download_count(rubygem)
   def stats_graph_meter(gem, count)
     gem.downloads * 1.0 / count * 100
   end
+
```

Comment:
Feels like the application layout could benefit from this quite a bit: https://github.com/olivierlacan/rubygems.org/blob/62fe96e0252bac0d21b5248d9092f7d5da85def0/app/views/layouts/application.html.erb#L37